smoke-tests/springboot/build.gradle

```diff
@@ -11,11 +11,18 @@ version = '0.0.1-SNAPSHOT'
 repositories {
   mavenCentral()
   jcenter()
+  // this is only needed for the working against unreleased otel-java snapshots
+  maven {
+    url "https://oss.jfrog.org/artifactory/oss-snapshot-local"
+    content {
+      includeGroup "io.opentelemetry"
+    }
+  }
 }
 
 dependencies {
   implementation 'org.springframework.boot:spring-boot-starter-web'
-  implementation 'io.opentelemetry:opentelemetry-extension-auto-annotations:0.7.0'
+  implementation 'io.opentelemetry:opentelemetry-extension-auto-annotations:0.10.0-SNAPSHOT'
```

Comment:
Ah thanks I got confused since muzzle was failing

Reply:
Ah wait - yeah the main point of the change wasn't the annotations but to pull in the newer API package (figured it's more idiomatic to just use the transitive dependency to manage that). I'm a bit confused why the smoke tests are passing - wouldn't you expect the instrumentation to get muzzled?

Reply:
`WithSpanAnnotationInstrumentation` uses shaded `Span.makeCurrent`, but doesn't rely on OpenTelemetry API in user class loader, so no muzzle problem (though we want to apply muzzle consistently across all instrumentation in a single module in the future https://github.com/open-telemetry/opentelemetry-java-instrumentation/issues/1274#issuecomment-707860035)

Oh, and this made me realized `@WithSpan` instrumentation is no longer in `external-annotations`, which makes me think maybe your change to `external-annotations.gradle` is not needed?

Reply:
Hmm - the `:muzzle` task was definitely failing in external-annotations though. Muzzle explicitly depends on the otel api, but should it be checking our shaded version instead?

Reply:
oh sorry, we should remove the annotations dep from that muzzle block in `external-annotations.gradle` since that module doesn't use it anymore (we should have removed it when we moved WithSpan annotation instrumentation from `external-annotations` to `opentelemetry-api-beta`), e.g.

```
muzzle {
  pass {
    group = "io.opentracing.contrib.dropwizard"
    module = "dropwizard-opentracing"
    versions = "[0.2.2,)"
  }
}
```

Reply:
Thanks!

Not sure if you meant this literally or figuratively - I tried adding muzzle for other libraries as well but dropwizard-opentracing seems to be the only one that passes for some reason.

Reply:
> Not sure if you meant this literally or figuratively

I use e.g. to mean a lot of things at once 😂